### PR TITLE
feat(cdn): init cds backend config

### PR DESCRIPTION
**Pull request by richardlt**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Review by fsamin — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14842.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✘  
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14842.1 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package and Analyze
  * Docker ✘  
  * Sonar ✔

File: engine/cdn/storage/cds/cds.go [hunk 1]
```diff
@@ -110,3 +107,13 @@ func (c *CDS) Status(_ context.Context) []sdk.MonitoringStatusLine {
 func (c *CDS) Remove(ctx context.Context, i sdk.CDNItemUnit) error {
 	return nil
 }
+
+func (c *CDS) Read(_ sdk.CDNItemUnit, r io.Reader, w io.Writer) error {
+	_, err := io.Copy(w, r)
+	return sdk.WithStack(err)
+}
+
+func (c *CDS) Write(_ sdk.CDNItemUnit, r io.Reader, w io.Writer) error {
```
Comment (sguiheux) [inline engine/cdn/storage/cds/cds.go:116]: maybe return nil ?  no write impementation on CDS backend

**Review by sguiheux — CHANGES_REQUESTED** [review]

**Review by yesnault — APPROVED** [review]

File: engine/cdn/storage/cds/cds.go (hunk 1 above)
Comment (richardlt) [reply engine/cdn/storage/cds/cds.go:116]: done

**Review by yesnault — APPROVED** [review]

**Review by sguiheux — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14885.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package and Analyze
  * Docker ✔ 
  * Sonar ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘

**Pull request merged**